source/sass/blocks/field.scss

```
@@ -79,15 +84,21 @@
   width: 100%;
   font-size: 16px;
   line-height: 18px;
-  padding-left: 19px;
-  padding-top: 10px;
+  padding-left: 15px;
+  padding-top: 9px;
   min-height: 150px;
-  margin-top: 30px;
+  margin-top: 36px;
+  margin-bottom: 22px;
 
   @media (min-width: $tablet-width) {
     margin-top: 70px;
     min-height: 174px;
     font-size: 20px;
     line-height: 30px;
+    margin-bottom: 66px;
```

Когда филдсет придет в себя, отступ надо будет пофиксить
```suggestion
    margin-bottom: 56px;
```